src/main.go

```diff
@@ -58,6 +58,10 @@ func main() {
 		// can be quite flakey. For this reason, we wrap most issues in a
 		// non-fatal error type.
 		if pluginConfig.FailBuildOnPluginFailure || runtimeerrors.IsFatal(err) {
+			// If timeout error, exit 75 (POSIX timer expired)
```

Comment:
No strong reason for using exit code 75. If there's a more appropriate one, happy to change it.